bin/civi-test-run

```diff
@@ -175,6 +144,14 @@ while [ -n "$1" ] ; do
     show_help
     exit 2
 
+  elif [ $1 = "all" ] ; then
+    if [ -f "$CIVI_CORE/tests/phpunit/E2E/AllTests.php" ]; then
```

Comment:
I like how this makes "all" do the Right Thing in more cases.

I'm less enthused about how it makes the order of arguments subtly more brittle -- eg these would behave differently:
 * `civi-test-run -b foo all` (includes or excludes E2E depending on the content of `foo`)
 * `civi-test-run all -b foo` (never runs E2E, regardless of version)

Reply:
hmmm

Reply:
Well, if you need per-build detection, then it can be fixed after the loop. It's a little less pretty but should be fine:

```bash
SUITES=
USE_ALL=
while [ -n "$1" ] ; do
  #...parsing...
  if [ "$1" = "all" ]; then
     USE_ALL=1
  fi
  #...parsing...
done

if [ "$USE_ALL"]; then
  # ...compute $SUITES...
fi
```